### PR TITLE
Fix instructions for merging bin / qemu

### PR DESCRIPTION
**Pull request by torkleyy**

### CODE REVIEW

File: src/tooling/simulating/qemu.md [hunk 1]
```diff
@@ -23,13 +23,13 @@ If you prefer to use [`espflash`][espflash], you can achieve the same result by
...
 cargo build --release
-espflash save-image --chip ESP32 --merge target/xtensa-esp32-espidf/release/<NAME> <OUTFILE>
+espflash save-image --chip esp32 --merge target/xtensa-esp32-espidf/release/<NAME> <OUTFILE>
 ```
 
 Now, run the image in QEMU:
 
 ```shell
-/path/to/qemu-system-xtensa -nographic -machine esp32 -drive file=<OUTFILE>,if=mtd,format=raw
+/path/to/qemu-system-xtensa -nographic -machine esp32 -drive file=<OUTFILE>,if=mtd,format=raw -m 4M
```
Comment (SergioGasquez) [inline src/tooling/simulating/qemu.md:32]: What does the `-m 4M` argument does? Flash size?

Comment (torkleyy) [reply src/tooling/simulating/qemu.md:32]: It allows using external PSRAM in the firmware. See https://github.com/espressif/esp-toolchain-docs/blob/main/qemu/esp32/README.md#adding-psram

Comment (SergioGasquez) [reply src/tooling/simulating/qemu.md:32]: Is that something that we would like to have included in the command that we showcase? We already have a link to the `esp-toolchain-docs` also, so user can check it and see the available args.

Comment (torkleyy) [reply src/tooling/simulating/qemu.md:32]: I thought think it's very common to use external RAM, this makes it easier to get started.

**Review by SergioGasquez — APPROVED** [review]

**Pull request merged**